### ISSUE TITLE
After update to Jedi 0.12.1 and Parso 0.3.1-- autocomplete crash

### ISSUE THREAD

**Issue by Alexey-T** [davidhalter/jedi#1204]
CudaText plugin Python Intel crashes now.
test file is:
```
MB_OK               = 0x00
MB_OKCANCEL         = 0x01
MB_ABORTRETRYIGNORE = 0x02
MB_YESNOCANCEL      = 0x03
MB_YESNO            = 0x04
```
caret y=3 x=2 (afer MB_).
autocomplete call gives crash.
```
Traceback (most recent call last):
  File "C:\Cudatext\py\cuda_python_intel\jedi\evaluate\compiled\subprocess\__init__.py", line 210, in _send
    is_exception, traceback, result = pickle_load(self._process.stdout)
  File "C:\Cudatext\py\cuda_python_intel\jedi\_compatibility.py", line 452, in pickle_load
    return pickle.load(file, encoding='bytes')
EOFError: Ran out of input

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\Cudatext\py\cuda_python_intel\__init__.py", line 39, in on_complete
    text = handle_autocomplete(*params)
  File "C:\Cudatext\py\cuda_python_intel\intel_work.py", line 10, in handle_autocomplete
    completions = script.completions()
  File "C:\Cudatext\py\cuda_python_intel\jedi\api\__init__.py", line 174, in completions
    completions = completion.completions()
  File "C:\Cudatext\py\cuda_python_intel\jedi\api\completion.py", line 98, in completions
    completion_names = self._get_context_completions()
  File "C:\Cudatext\py\cuda_python_intel\jedi\api\completion.py", line 173, in _get_context_completions
    completion_names = list(self._get_keyword_completion_names(allowed_transitions))
  File "C:\Cudatext\py\cuda_python_intel\jedi\api\completion.py", line 212, in _get_keyword_completion_names
    yield keywords.KeywordName(self._evaluator, k)
  File "C:\Cudatext\py\cuda_python_intel\jedi\api\keywords.py", line 28, in __init__
    self.parent_context = evaluator.builtins_module
  File "C:\Cudatext\py\cuda_python_intel\jedi\evaluate\cache.py", line 40, in wrapper
    rv = function(obj, *args, **kwargs)
  File "C:\Cudatext\py\cuda_python_intel\jedi\evaluate\__init__.py", line 115, in builtins_module
    return compiled.get_special_object(self, u'BUILTINS')
  File "C:\Cudatext\py\cuda_python_intel\jedi\evaluate\compiled\__init__.py", line 27, in get_special_object
    evaluator.compiled_subprocess.get_special_object(identifier)
  File "C:\Cudatext\py\cuda_python_intel\jedi\evaluate\compiled\subprocess\__init__.py", line 94, in wrapper
    kwargs=kwargs,
  File "C:\Cudatext\py\cuda_python_intel\jedi\evaluate\compiled\subprocess\__init__.py", line 166, in run
    return self._send(id(evaluator), function, args, kwargs)
  File "C:\Cudatext\py\cuda_python_intel\jedi\evaluate\compiled\subprocess\__init__.py", line 221, in _send
    stderr,
jedi.api.exceptions.InternalError: The subprocess C:\Cudatext\cudatext.exe has crashed (EOFError('Ran out of input',), stderr=b'').
```

Win10, Python 3.5 x64 DLL embedded

**Comment by Alexey-T** [comment]
@OlehL thinks that Jedi want to run new subprocess of Python, which path it gets from  sys.executable, but for Emdedded DLL this path is path to CudaText app.

**Comment by davidhalter** [comment]
Is this still an issue?

**Comment by Alexey-T** [comment]
Fixed in 0.13
